networking/ovn_kubernetes_network_provider/migrate-from-kuryr-sdn.adoc

```diff
@@ -7,7 +7,6 @@ include::_attributes/common-attributes.adoc[]
 toc::[]
 
 :FeatureName: Migration from Kuryr to OVN-Kubernetes
-include::snippets/technology-preview.adoc[]
```

Comment:
You can delete L9, too--the snippet uses the `FeatureName` attribute.

Reply:
ack.